sympy/combinatorics/coset_table.py

```diff
@@ -856,22 +881,32 @@ def modified_define(self, alpha, x):
         Define a function p_p from from [1..n] to A* as
         an additional component of the modified coset table.
 
-        Input: \alpha \in \Omega, x \in A*
+        Parameters
+        ==========
+
+        \alpha \in \Omega
+        x \in A*
```

Comment:
See [sample docstring](https://github.com/sympy/sympy/blob/master/sympy/calculus/singularities.py#L28-L43) illustrating the way parameters are formatted.

Reply:
Yes, this was a sort of intermediate approach. Add heading and split the line, but not go all the way and actually understand what things means.

Reply:
I noted that in that docstring the examples came last. In most other they come before Notes, See Also and References (with References most often last). Is there a specified order? I've tried to end with See Also and References, in that order.

Reply:
The docstring sections are generally written in the order specified at https://github.com/sympy/sympy/wiki/Writing-documentation#notes.

Reply:
Thanks for clarifying. That is consistent with my impression, so that's good. Not that I can guarantee that all my cleanup PRs really follow it, but if I modified something it is at least more close to it. In the future it can of course improve further.